indico/modules/events/surveys/controllers/management/questionnaire.py

```diff
@@ -188,14 +281,21 @@ def _process(self):
         except KeyError:
             raise NotFound
 
-        form = field_cls.config_form()
+        clone_id = request.args.get('clone')
+        form = None
+        if clone_id is not None:
+            try:
+                question_to_clone = SurveyQuestion.query.with_parent(self.survey).filter_by(id=int(clone_id)).one()
+                form = question_to_clone.field.config_form(
+                        obj=FormDefaults(question_to_clone, **question_to_clone.field.copy_field_data()))
+            except (NoResultFound, ValueError):
```

Comment:
still not a fan of such a generic ValueError except block. the only one you really want to watch is in `int(clone_id)`